gunicorn/config.py

```diff
@@ -1811,3 +1819,16 @@ class PasteGlobalConf(Setting):
 
         .. versionadded:: 20.0
         """
+
+class SpawnWorkerSleepTime(Setting):
+    name = "spawn_worker_sleep_time"
+    section = "Worker Processes"
+    cli = ['--spawn-worker-sleep-time']
```

Comment:
small nitpicking but this quite a long option name. Maybe something shorter would be better? like `--spawn-worker-sleep` or something more native?